apps/chronicles/screens/components/SwitchCellScreen.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import {
+    Cell,
+    EDSStyleSheet,
+    Spacer,
+    Typography,
+    useStyles,
+    Switch,
+} from "@equinor/mad-components";
+import { ScrollView, View } from "react-native";
+
+export const SwitchCellScreen = () => {
+    const styles = useStyles(themeStyles);
+
+    const [activeSwitch1, setActiveSwitch1] = useState(false);
+    const [activeSwitch2, setActiveSwitch2] = useState(true);
+    const [activeSwitch3, setActiveSwitch3] = useState(false);
+    const [disabledSwitch, setDisabledSwitch] = useState(true);
```

Comment:
Maybe this should be removed in favor of a `true` where needed in code 😠

Reply:
yes i agree 😡